web/src/routes/MyPages/ProfilePage.tsx

```diff
@@ -114,30 +109,28 @@ const ReplacePagetext = styled.span`
   color: ${(props) => props.theme.colors.default};
 `;
 
-const CompleteButton = styled.div`
-  margin-left: auto;
+const PurchaseButton = styled.div`
+  margin: 2vw;
   display: flex;
-  padding: 1.5vh;
+  padding: 1.3vh;
+  border: none;
+  border-radius: 10px;
   font-size: 1.8vh;
-  font-weight: 700;
+  font-weight: 600;
   color: white;
   background-color: ${(props) => props.theme.colors.primary};
   border: none;
   border-radius: ${(props) => props.theme.borderRadius};
   cursor: pointer;
   align-items: center;
-  justify-content: center;
-  white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
 
   &:hover {
     background-color: ${(props) => props.theme.colors.prhover};
   }
 `;
 
-// FIXME: To API data
 const initalState = {
+  point: '0P',
```

Comment:
Point should be fetched as `string` type from API. But the letter `P` should be added from client. Please detach `P` letter.

**Reason for the type of point is not `number` but `string`**:
In case of capturing negative value from API which would be used to create a survey or exchanging to a gifticon.